<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewPoolProvisionDialog</class>
 <widget class="QDialog" name="NewPoolProvisionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>363</width>
    <height>367</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_protocol">
     <item>
      <widget class="QLabel" name="label_protocol">
       <property name="text">
        <string>Protocol</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_protocol"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget_tokens">
     <row>
      <property name="text">
       <string>-</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>-</string>
      </property>
     </row>
     <row>
      <property name="text">
       <string>+</string>
      </property>
     </row>
     <column>
      <property name="text">
       <string>Token</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quantity</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Weight</string>
      </property>
     </column>
     <item row="2" column="0">
      <property name="text">
       <string/>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
     <item row="2" column="1">
      <property name="text">
       <string/>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
     <item row="2" column="2">
      <property name="text">
       <string/>
      </property>
      <property name="flags">
       <set>NoItemFlags</set>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_poolFee">
     <item>
      <widget class="QLabel" name="label_poolFee">
       <property name="text">
        <string>Pool fee (%)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBox_poolFee">
       <property name="currentIndex">
        <number>-1</number>
       </property>
       <item>
        <property name="text">
         <string>0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.25</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.5</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>0.75</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>1</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_slippageController">
     <item>
      <widget class="QLabel" name="label_slippageController">
       <property name="text">
        <string>Slippage controller</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit_slippageController"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_provide">
     <property name="text">
      <string>Provide</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
